dimagi/pages/templates/sections/sectors/disease-treatment/news.html

```diff
@@ -0,0 +1,146 @@
+                A Peace Corps volunteer explains how CommCare aided in a patient data collection
+                program in Cameroon focused on HIV treatment around World AIDS Day.
+              {% endblocktrans %}
+            </p>
+            <p class="pad-top-small-at-large pad-top-small-at-extra-large">
+              <a href="https://www.dimagi.com/blog/why-peace-corps-cameroon-is-leveraging-mobile-data-collection-in-the-fight-against-hiv-aids/"
+                 class="link-gray read-more"
+                 target="_blank">
+                {% blocktrans %}
+                  Read More &gt;
+                {% endblocktrans %}
+              </a>
+            </p>
+          </div> {# .pad-line #}
+        </div> {# .white-bg #}
+      </div> {# .column #}
+      <div class="column">
+        <div class="pad-bottom-base white-bg">
+          <div class="image blog2" data-lazybg="true"></div>
+          <div class="pad-line">
+            <h5>
+              <strong>
+                {% blocktrans %}
+                  USAID Awards Dimagi & Partners a Grand Challenge to Combat Zika and Future Outbreaks
+                {% endblocktrans %}
+              </strong>
+            </h5>
+            <p>
+              {% blocktrans %}
+                USAID announced that Dimagi and partners have been awarded a Combating Zika and Future
+                Threats Grand award — joining a select group of innovators focused on curbing infectious
+                disease worldwide.
+              {% endblocktrans %}
+            </p>
+            <p>
+              <a href="https://www.dimagi.com/blog/usaid-awards-dimagi-partners-a-grand-challenge-to-combat-zika-and-future-outbreaks/"
+                 class="link-gray read-more"
+                 target="_blank">
+                {% blocktrans %}
+                  Read More &gt;
+                {% endblocktrans %}
+              </a>
+            </p>
+          </div> {# .pad-line #}
+        </div> {# .white-bg #}
+      </div> {# .column #}
+      <div class="column">
+        <div class="pad-bottom-base white-bg">
+          <div class="image blog3" data-lazybg="true"></div>
+          <div class="pad-line">
+            <h5>
+              <strong>
+                {% blocktrans %}
+                  Lessons from Contact Tracing During the Ebola Outbreak
+                {% endblocktrans %}
+              </strong>
+            </h5>
+            <p class="pad-bottom-base-at-large">
+              {% blocktrans %}
+                Three pieces of advice in project and app design from a member of our Global Services
+                 team who spent a year working in Liberia, Sierra Leone, and Guinea on mobile tech to
+                 combat Ebola.
+              {% endblocktrans %}
+            </p>
+            <p class="pad-top-small-at-large pad-top-base-at-extra-large">
+              <a href="https://www.dimagi.com/blog/contact-tracing-during-the-ebola-outbreak-notes-from-the-field/"
```

Comment:
use {% url 'blog_post' 'contact-tracing-during-the-ebola-outbreak-notes-from-the-field' %}